From 328b5f09e171716f28a81be4f07ca8f3cea55942 Mon Sep 17 00:00:00 2001
From: Yotam Ben Dosa <ybendosa@habana.ai>
Date: Thu, 26 Dec 2019 15:59:50 +0200
Subject: [PATCH] [SW-0] openbmc: install-hls: deploy image to tmpfs

The mainline BMC maps to root partition to the flash storage, which is
not big enough to containt the update image, so tmpfs will be used
instead, as it's memory mapped.

Change-Id: Ieb26e6d9169a3ed356d06cdebe24c519aef3fb9e
Signed-off-by: Yotam Ben Dosa <ybendosa@habana.ai>
---

diff --git a/package/openbmc/install-hls.sh b/package/openbmc/install-hls.sh
index 6d60148..62c0dab 100755
--- a/package/openbmc/install-hls.sh
+++ b/package/openbmc/install-hls.sh
@@ -66,5 +66,5 @@
 BMC_ADDRESS=${1}
 
 sshpass -p $BMC_ROOT_PASSWORD scp -o StrictHostKeyChecking="no" $BMC_IMAGE \
-	root@$BMC_ADDRESS:
-sshpass -p $BMC_ROOT_PASSWORD ssh root@$BMC_ADDRESS "update_flash.sh -r"
+	root@$BMC_ADDRESS:/tmp
+sshpass -p $BMC_ROOT_PASSWORD ssh root@$BMC_ADDRESS "pushd /tmp && update_flash.sh -r"
